src/main/java/com/redhat/rhjmc/containerjfr/core/ContainerJfrCore.java

```diff
@@ -52,11 +56,22 @@
 public class ContainerJfrCore {
     private ContainerJfrCore() {}
 
-    public static void initialize() throws CoreException {
+    public static void initialize()
+            throws CoreException, SecurityException, NullPointerException, IOException {
         System.setProperty(
                 "org.openjdk.jmc.common.security.manager",
                 SecurityManager.class.getCanonicalName());
-        LogManager.getLogManager().reset();
+
+        String jmcLoggerLevelProperty = "org.openjdk.jmc.level=" + Level.OFF.getName();
+        String vertxLoggerLevelProperty = "io.vertx.level=" + Level.OFF.getName();
+
+        String config = jmcLoggerLevelProperty + "\n" + vertxLoggerLevelProperty;
+
+        LogManager.getLogManager()
+                .updateConfiguration(
+                        new ByteArrayInputStream(config.getBytes(StandardCharsets.US_ASCII)),
```

Comment:
This `ByteArrayInputStream` should be closed just as a matter of code cleanliness, IMO, even though closing this type of stream does nothing. Please wrap it in a try-with-resources.